titiler/pgstac/main.py

```diff
@@ -33,6 +37,23 @@
 app = FastAPI(title=settings.name, version=titiler_pgstac_version)
 
 
+def custom_openapi():
+    if app.openapi_schema:
+        return app.openapi_schema
+    openapi_schema = get_openapi(
+        title="TiTiler PgSTAC",
+        version=titiler_pgstac_version,
+        description="A lightweight STAC API implementation using PostgreSQL as a backend.",
+        routes=app.routes,
+        openapi_version="3.0.1",
+    )
+    app.openapi_schema = openapi_schema
+    return app.openapi_schema
+
+
+app.openapi = custom_openapi
```

Comment:
Is this also part of the `proxy header` work?

Reply:
oh sorry, that whole block I needed to set openapi version to 3.0.1 explicitly to work with Azure APIM, can exclude that part (L40-L54)